datafusion/optimizer/tests/integration-test.rs

```diff
@@ -195,9 +195,10 @@ fn between_date64_plus_interval() -> Result<()> {
     WHERE col_date64 between '1998-03-18T00:00:00' AND cast('1998-03-18' as date) + INTERVAL '90 days'";
     let plan = test_sql(sql)?;
     let expected =
-        "Projection: COUNT(Int64(1))\n  Aggregate: groupBy=[[]], aggr=[[COUNT(Int64(1))]]\
+        "Aggregate: groupBy=[[]], aggr=[[COUNT(Int64(1))]]\
+        \n  Projection: test.col_int32\
         \n    Filter: test.col_date64 >= Date64(\"890179200000\") AND test.col_date64 <= Date64(\"897955200000\")\
-        \n      TableScan: test projection=[col_date64]";
+        \n      TableScan: test projection=[col_int32, col_date64]";
```

Comment:
One extra column is cause by `aggr=[[COUNT(Int64(1))]]`.
When aggr don't use any column, pushdown_projection will push first column of child schema.

Reply:
would it be possible to not push a column down if there was already a column that was scanned (e.g. in this case use col_date64?)

Reply:
> would it be possible to not push a column down if there was already a column that was scanned (e.g. in this case use col_date64?)

I have resolved it !🚀